packages/web-components/fast-element/src/template-compiler.ts

```diff
@@ -7,14 +7,46 @@ import { ExecutionContext, Binding } from "./observation/observable";
 type InlineDirective = Directive & {
     targetName?: string;
     binding: Binding;
-    targetAtContent();
+    targetAtContent(): void;
 };
 
-const compilationContext = { locatedDirectives: 0, targetIndex: -1 };
+class CompilationContext {
+    public targetIndex!: number;
+    public behaviorFactories!: BehaviorFactory[];
+    public directives: ReadonlyArray<Directive>;
+
+    public addFactory(factory: BehaviorFactory) {
+        factory.targetIndex = this.targetIndex;
+        this.behaviorFactories.push(factory);
+    }
+
+    public captureContentBinding(directive: BindingDirective): void {
```

Comment:
This method has an explicit `void` return type, though the other methods in the class don't. Is this intentional?

Reply:
Nope 🤣 Will fix.